en/components/themes/misc/printing-styles.md

```diff
@@ -33,6 +29,22 @@ core($print-layout: false)
 @include theme($default-palette);
 ```
 
-Note that even if you have a colorful grid it will appear black and white in order to be as clean and simple as possible and to reduce the use of ink. All the clickable elements in the grid will be omitted since they don't make sense on a paper
+Since **v13.2** we decide not to hide any component by default since we don't know what parts you want to be visible on paper, we leave that for you to decide.   
```

Comment:
```suggestion
Since **v13.2** we decided not to hide any component by default since we don't know what parts you want to be visible on paper, we leave that for you to decide.   
```